.github/workflows/check-translations-and-deserialize.yml

```diff
@@ -56,6 +60,9 @@ jobs:
         uses: peter-evans/create-pull-request@v3
         with:
           token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
-          title: Add new translations
+          title: Updated translations - ${{ steps.date.outputs.date }}
```

Comment:
Looks like the date is missing in the title: https://github.com/newrelic/docs-website/pull/667

Reply:
yes! addressing this now :)

Reply:
also adding it to be deleted afterwards